source/learn.html.erb

```diff
@@ -12,9 +12,9 @@ responsive: true
   <p>Are you getting started on your Ember.js adventure? Check out our official reading material: </p>
 
   <ul>
-    <li><a href="https://guides.emberjs.com/v2.9.0/getting-started/quick-start/"><strong>Quick start</strong> <i class="icon-link-ext"></i></a>: An easy breazy introduction to the framework.</li>
-    <li><a href="https://guides.emberjs.com/v2.5.0/tutorial/ember-cli/"><strong>Tutorial</strong> <i class="icon-link-ext"></i></a>: Follow our tutorial to build, and deploy, your first Ember.js application!</li>
-    <li><a href="https://guides.emberjs.com/"><strong>Guides</strong> <i class="icon-link-ext"></i></a>: Whether you're just getting started and want to get familiar with Ember, or you're looking to refresh your knowledge on a certain feature, the Guides are the place for you.</li>
+    <li><strong>Quick start</strong> <a href="https://guides.emberjs.com/v2.9.0/getting-started/quick-start/"><i class="icon-link-ext"></i></a>: An easy breazy introduction to the framework.</li>
+    <li><strong>Tutorial</strong> <a href="https://guides.emberjs.com/v2.5.0/tutorial/ember-cli/"><i class="icon-link-ext"></i></a>: Follow our tutorial to build, and deploy, your first Ember.js application!</li>
+    <li><strong>Guides</strong> <a href="https://guides.emberjs.com/"><i class="icon-link-ext"></i></a>: Whether you're just getting started and want to get familiar with Ember, or you're looking to refresh your knowledge on a certain feature, the Guides are the place for you.</li>
```

Comment:
Seems like it'd be good to use the `current_version` helper for these links as well so we don't have to come back and update regularly ... (see it in action here: https://github.com/emberjs/website/blob/master/source/about.html.erb#L40)